tests/heif/bug788.c

```diff
@@ -0,0 +1,43 @@
+/**
+ * Bug 788 stride not implemented.
+ */
+
+#include "gd.h"
+#include "gdtest.h"
+
+#include <libheif/heif.h>
+
+int main () {
+	FILE *fp;
+	gdImagePtr in;
+    gdImagePtr dst;
```

Comment:
Nit: mixed indentation (spaces vs. tabs) (also further down in the test file)

Reply:
fixed WS, thanks :)